doc/index.rst

```diff
@@ -44,6 +44,7 @@ Core Lightning Documentation
    lightning-checkmessage <lightning-checkmessage.7.md>
    lightning-cli <lightning-cli.1.md>
    lightning-close <lightning-close.7.md>
+   lightning-commando-listrunes <lightning-commando-listrunes.7.md>
```

Comment:
This commit should probably contain all the listrunes code (and have the previous one just save the runes?).  Since we rewrite half of it here!